### ISSUE TITLE
[ENHANCEMENT] Support customizing PVC annotation in Zeebe

### ISSUE THREAD

**Issue by aabouzaid** [camunda/camunda-platform-helm#1230]
**Describe the use case:**

As a user, I need to set annotation for Zeebe PVC.

**Describe the enhancement/feature:**

Need to set annotation to Zeebe `volumeClaimTemplates` which should be straightforward like the rest of `zeebe.pvc*` values.

**Desired outcome and acceptance tests:**

- Support customizing PVC annotation in Zeebe via  `zeebe.pvcAnnotations`.

Links:
- https://jira.camunda.com/browse/SUPPORT-19925
